### DISCUSSION TITLE
How to sign in Tidy3d?

### DISCUSSION THREAD

**Discussion by lamph123** [flexcompute/tidy3d#209] [General]
Hello guys,

How can I sign in to Tidy3d?

THank you so much!

**Comment by tylerflex** [comment]
Hi @lamph123 
Sorry for the delay.  You can sign up for our waitlist here: https://my.simulation.cloud/signup-waiting?ref=https://tidy3d.simulation.cloud/. We will reach out in the next few days or weeks with some signup information.
